notebooks/official/tensorboard/tensorboard_vertex_ai_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,829 @@
+{
```

Comment:
<p>I would move this to the install section. Drop the text explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>